moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -1127,9 +1128,4 @@ void ServoCalcs::setPaused(bool paused)
   paused_ = paused;
 }
 
-void ServoCalcs::changeRobotLinkCommandFrame(const std::string& new_command_frame)
-{
-  parameters_->robot_link_command_frame = new_command_frame;
-}
-
```

Comment:
@AndyZe This is why I removed this function.  This is no longer a constant parameter.  This parameter needs to be handled separately from the rest of the parameters if it can be changed during runtime from the rest of the parameters.

What if this changes between the use of it here: https://github.com/ros-planning/moveit2/blob/bbe859ee6c2bd35c9ba01f666ff9e9a8a4915e1f/moveit_ros/moveit_servo/src/servo_calcs.cpp#L309-L314 and here: https://github.com/ros-planning/moveit2/blob/bbe859ee6c2bd35c9ba01f666ff9e9a8a4915e1f/moveit_ros/moveit_servo/src/servo_calcs.cpp#L463

Should the value of this be changed only between runs of the servo calcs loop or can it be changed while it is being used?

Reply:
ideally it could be changed at any time, not just while Servo is paused